content/app/github/commit-page-helper.js

```diff
@@ -26,11 +26,18 @@ define("content/app/github/commit-page-helper", function(require, exports, modul
 				var files = CommitPageHelper.getAllFiles();
 
 				var file = null;
+				var maxArea = 0;
```

Comment:
In order to make this behaviour deterministic from user point of view, I think that we need to mark current active file , the one which takes the most space on the screen. But also, I think that we need to improve this strategy, since it has some problems. For example, lets imagine large screen, and three files. First and last are really big and the one in the middle is really small. With this approach, a user could never 'select' the small one. My conclusion is that we definitively need to put distance of file header from top of the screen in the calculation, or something similar.